scripts/tracking/test.py

```diff
@@ -0,0 +1,115 @@
+""" SiamRPN test """
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+import argparse
+import os
+import numpy as np
+from mxnet import gpu
+from gluoncv.model_zoo.siamrpn.siam_net import SiamrpnNet as ModelBuilder
+from gluoncv.utils.siamrpn_tracker import SiamRPNTracker as build_tracker
+from gluoncv.data.otb.tracking import OTBTracking as OTBDataset
+from gluoncv.utils.siamrpn_tracker import get_axis_aligned_bbox
+from gluoncv.utils.filesystem import try_import_cv2
+
+def parse_args():
+    """parameter test."""
+    parser = argparse.ArgumentParser(description='siamrpn tracking test result')
+    parser.add_argument('--dataset', default='OTB2015', type=str, help='dataset name')
+    parser.add_argument('--dataset_root', type=str, default='~/.mxnet/datasets/OTB2015', help='dataset_root')
+    parser.add_argument('--model_path', type=str, help='path of models to eval')
+    parser.add_argument('--results_path', type=str, help='results path')
+    parser.add_argument('--video', default='', type=str,
+                        help='eval one special video')
+    parser.add_argument('--vis', action='store_true',
+                        help='whether visualzie result')
+    parser.add_argument('--mode', type=str, default='hybrid',
+                        help='mode in which to train the model.options are symbolic, hybrid')
+    opt = parser.parse_args()
+    return opt
+
+def main():
+    """SiamRPN test.
+    function
+    ----------
+        record the output of the model. The output information of each video is recorded in the txt
+    corresponding to the video name.
+        if you want to evaluation, you need to python benchmark.py according to txt of text result.
+        Currently only supports test OTB 2015 dataset
+    Parameters
+    ----------
+    dataset_root : str, default '~/mxnet/datasets/OTB2015'
+                   Path to folder test the dataset.
+    model_path :   str, Path of test model .
+    results_path:  str, Path to store txt of test reslut .
+    """
+    opt = parse_args()
+    # dataloader
+    dataset = OTBDataset(name=opt.dataset, dataset_root=opt.dataset_root, load_img=False)
+    # network
+    model = ModelBuilder()
```

Comment:
I forgot to mention, here should use get_model from `model_zoo.py` instead of simply calling a function.